scripts/lis_hotfix_pipeline/CreateBuildMachines.ps1

```diff
@@ -0,0 +1,167 @@
+﻿param (
+    $DistroKernelVersions="",
+    $VHDSourceStorageAccount = "",
+    $BootDiagnosticStorageAccount = "",
+    $secretsFile = "",
+    $LinuxUsername = "",
+    $LinuxPassword = ""
+)
+
```

Comment:
better to have comments for the usage of ps1 files or method

Reply:
Added description in the beginning of the script.